WebHangman.js

```diff
@@ -0,0 +1,128 @@
+  let gameStatus = document.querySelector('#game-status')
+  
+  let myHiddenWord = document.querySelector('#hiddenword')
+  let myForm = document.querySelector('#myForm')
+
+  let numberOfGuesses = document.querySelector('#guess-counter')
+  numberOfGuesses.innerText = '6 guesses left'
+  let tries = 6
+  
+  // get a random word from the words array
+   let randomWord = getRandomWord(words);
+
+   //create hiddenword, ie replace char with underscores
+   let hiddenWord = createHiddenWord(randomWord);
+   myHiddenWord.innerText = hiddenWord
+
+   console.log(hiddenWord)
+   console.log(randomWord)
+   
+   //create usedLetters array
+   usedLetters = []
+  
+  // Gameplay: after each click functions run to check letter and change DOM accordingly
+
+  myForm.addEventListener('submit',(event)=> {
+
+    event.preventDefault();
+    let letterInput = document.querySelector('#letterinput').value.toLowerCase()
+    let letterGuessed = document.querySelector('#guess-letters')
+
+    // check if letterguess is in hiddenword
+
+       //makes sure input is valid like salad
+     if (letterInput === '' || letterInput.length !== 1 || letterInput > -Infinity){
+      gameStatus.innerText= 'Not a valid guess';
+      document.getElementById('myForm').reset()
+
+      //replace innerText
+    } else if (isLetterInWord(letterInput, randomWord, hiddenWord)){
+      hiddenWord = replaceMatches(letterInput, randomWord, hiddenWord)
+      myHiddenWord.innerText= hiddenWord
+      gameStatus.innerText = 'Correct Guess'
+      document.getElementById('myForm').reset()
+      usedLetters.push(letterInput)
+
+
+        // if no: lose a life
+    } else if(!isLetterInWord(letterInput, randomWord, hiddenWord)){
+      gameStatus.innerText = 'Incorrect guess'
+      tries = tries -1 
+      usedLetters.push(letterInput)
+      numberOfGuesses.innerText = tries + ' guesses/guess left'
+      document.getElementById('myForm').reset()
+    }
+    
+    letterGuessed.innerText = usedLetters
+    console.log(usedLetters)
+    
+    // We're in the end game now
+    if (tries === 0){
+      gameStatus.innerText = `EXODIAAAAAAAAAAAA. The word you were looking for was: ${randomWord}`
+      window.alert('Game over, you lose')
+      document.getElementById('myForm').reset()
+      // location = location
+
+    } else if (hiddenWord.join('') === randomWord){
+      gameStatus.innerText = `Congrats you found the hidden word .....  ${randomWord} `
+      usedLetters.push(letterInput)
+      document.getElementById('myForm').reset()
+      window.alert("You won!!!!")
+      myHiddenWord.innerText = hiddenWord.join('')
+    }
+  })   
+   //PUT REFRESH FUNCTION BUTTON OUTSIDE SUBMIT EVENT
+})
+
+// Defining functions 
+ 
+  const newGame = () => location = location
```

Comment:
What does this function do? I don't see newGame being called anywhere. Maybe I missed it somewhere?